packages/blockchain-wallet-v4-frontend/src/modals/Onboarding/LinkFromExchangeAccount/template.success.tsx

```diff
@@ -15,50 +23,264 @@ const getIcon = tier => {
   }
 }
 
-const Success: React.FC<Props & { close: () => void }> = props => {
+const getStatus = tier => {
+  switch (tier) {
+    case 1:
+      return 'Silver Verified'
+    case 2:
+      return 'Gold Verified'
+    default:
+      return ''
+  }
+}
+
+const IconWrapper = styled.div`
+  display: flex;
+  width: 100%;
+  justify-content: flex-start;
+`
+
+const InfoWrapper = styled.div`
+  width: 100%;
+  display: flex;
+  margin-bottom: 32px;
+`
+const InfoText = styled(Text)`
+  width: 100%;
+  display: flex;
+  text-align: left;
+  a {
+    color: ${props => props.theme.blue600};
+    text-decoration: none;
+  }
+`
+const ImageWrapper = styled.div`
+  width: 40px;
+  justify-content: flex-start;
+  display: flex;
+  flex-direction: column;
+`
+
+const ListText = styled(Text)`
+  text-align: left;
+  margin-left: 18px;
+  margin-top: 8px;
+  span {
+    text-indent: -12px;
+    display: inline-block;
+    padding-left: 12px;
+  }
```

Comment:
whole magic is done here we do alignment of text to move every next line to be aligned with text from 1st line where we have dash